src/emulator/gxm/include/gxm/functions.h

```diff
@@ -32,15 +32,16 @@ GLenum attribute_format_to_gl_type(SceGxmAttributeFormat format);
 bool attribute_format_normalised(SceGxmAttributeFormat format);
 void set_uniforms(GLuint program, const SceGxmContext &context, const MemState &mem);
 void flip_vertically(uint32_t *pixels, size_t width, size_t height, size_t stride_in_pixels);
-GLenum translate_blend_func(SceGxmBlendFunc src);
-GLenum translate_blend_factor(SceGxmBlendFactor src);
+GLenum translate_blend_func(const SceGxmBlendFunc& src);
```

Comment:
Isn't `SceGxmBlendFunc` an integer? It's OK to pass by value.

Unless the goal is to forward declare them and remove `#include <psp2/gxm.h>` from this file?

Reply:
Yeah all of those args I changed are enums. Isn't passing enums by reference ok? I read that the compiler will do the same thing anyway (will actually pass by value).
Anyway, I'll change all of them back to by value.